components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -80,8 +81,28 @@ export class CreateWorkspace extends React.Component<CreateWorkspaceProps, Creat
     }
 
     else if (result?.existingWorkspaces) {
-      // FIXME Force create
-      statusMessage = <div className="text-base text-gray-400">Existing workspaces:<ul>{result.existingWorkspaces.map(w => <li>→ <a className="text-blue" href={w.latestInstance?.ideUrl}>{w.workspace.id}</a></li>)}</ul></div>;
+      statusMessage = <Modal visible={true} closeable={false}>
+        <h3>Running Workspaces</h3>
+        <div className="border-t border-b border-gray-200 mt-2 py-2">
+          <p className="mt-1 mb-2 text-base">You already have running workspaces with the same context. You can open an existing one or open a new workspace.</p>
+          <>
+            {result?.existingWorkspaces?.map(w =>
+              <a href={w.latestInstance?.ideUrl} className="rounded-xl hover:bg-gray-200 flex p-3 my-1">
+                <div className="w-full">
+                  <p className="text-base text-black font-bold">{w.workspace.id}</p>
+                  <p>{w.workspace.contextURL}</p>
+                </div>
+                <div className="flex">
+                  <button className="border-green-dark bg-gitpod-green px-3 py-1 my-auto">Open</button>
```

Comment:
**issue**: We're still missing the hover state for the primary buttons. What do you think of using `#6E960D` for the background on hover?

Reply:
Hmm, sure thing, but I wouldn't know where to put it.

- Call it `gitpod-green-dark`?
- Entirely replace the 3 green shades from MAD?

Reply:
Since they've removed the green from their palette, let's replace green with `#7BA80F`, green dark with `#6E960D`, and replace green light with `#9DD714`.WDYT?

Reply:
Sounds great! Many thanks 🙏

Reply:
@gtsiolis But what about the previous Green Dark #467A45 used for button borders?

Reply:
Oh, it should be fine but let's replace that with `#53710A` to make the difference as small as possible. WDYT?

Reply:
Actually, as discussed in a call, let's switch to Tailwind's `colors.lime` [color palette](https://tailwindcss.com/docs/customizing-colors). 🍋 🥃 💫